docs/proposals/scheduler/agent.proto

```diff
@@ -0,0 +1,46 @@
+syntax = "proto3";
+
+message Empty {}
+
+
+// ------------------
+// Lifecycle
+// ------------------
+
+message ConnectionRequest {
+    // agent certificate here
+}
```

Comment:
You want to rely on the transport protocol to establish a secure connection. TLS for example.
The connection request may include an agent UUID.

Reply:
Thanks for the clarification. We'll also [take the suggestion](https://github.com/dev-sys-do/orka/pull/2/commits/dae37730c6d56acaa1cfbe39fe77b81d9714271b#diff-f9016a684c4143142c7c2e30d94d45c0ad67414d5d3c8fee1e5fc05320aa4ad4R11).